packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -362,7 +361,7 @@ function Panel({
       // Only trigger filter change event if there are no items
       if (items.length === 0) {
         // Trigger filter event to populate panel on first open
-        onFilterChange(filterValue, null)
+        onFilterChange(filterValue, undefined)
```

Comment:
```suggestion
        onFilterChange(filterValue)
```

Would this be an equivalent or no? (I assumed it would be but now I'm double checking myself lol)